deploy-cloud-run/action.yml

```diff
@@ -45,11 +63,134 @@ runs:
       env:
         IMAGE: ${{ inputs.image }}
         IMAGE_INTERNAL: ${{ inputs.image_internal }}
+    - name: Parse test_url and retrieve the path
+      id: parse_url
+      if: ${{ inputs.test_url != '' }}
+      run: |
+        extract_path() {
+            local url="$1"
+            # Remove scheme (http:// or https://) and domain, keep only the path
+            local path=$(echo "$url" | sed -E 's|https?://[^/]+||')
+            # If path is empty, set to "/"
+            if [[ -z "$path" ]]; then
+                path="/"
+            fi
+            echo "url_path=$path" >> "${GITHUB_OUTPUT}"
+        }
+        extract_path "${{ inputs.test_url }}"
```

Comment:
@rot1024 this step is used to retrieve the path and simulate testing on blue instance on line 136